README.md

```diff
@@ -114,6 +114,7 @@ variable.
 Valid values for `type` are:
 
   - `ethernet`
+  - `infiniband`
```

Comment:
The subject of the commit is wrong: "infiniband is a valid connection <<TYPE>> as well".

`git commit --amend ...`

Maybe add the tag "doc: " as a prefix for each subject line of all these commits?

Reply:
you mean "infiniband is a valid connection type as well"?

Reply:
Exactly: `connection "type"`. I used \< which caused a loss in markdown.

Reply:
I rebased the branch with adding the tag as you suggest.